graphics/rendering/blender/blender.def

```diff
@@ -1,40 +1,39 @@
-Bootstrap: docker
+Bootstrap: library
 From: ubuntu:16.04
 
 %post
-    apt-get update
-    apt-get -y install blender blender-data
+apt-get update
+apt-get -y install blender blender-data
 
 %help
-    singularity run blender.sif [scene file] [output directory] <frame | start frame:end frame>
+singularity run blender.sif [scene file] [output directory] <frame | start frame:end frame>
```

Comment:
To let them know this is a command, might need to add `$` so.. `$ singularity run blender.sif ...`

Reply:
:+1: